<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Level</class>
 <widget class="PlayerPlugin" name="GUI_Level">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>51</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Level</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="0" column="9">
    <widget class="QLabel" name="lab_l_45">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lab_l_0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="lab_r_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lab_l_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="18">
    <widget class="QLabel" name="lab_r_9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="lab_l_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="lab_l_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lab_r_0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QLabel" name="lab_l_55">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="14">
    <widget class="QLabel" name="lab_r_7">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="lab_r_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="18">
    <widget class="QLabel" name="lab_l_9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QLabel" name="lab_r_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="16">
    <widget class="QLabel" name="lab_r_8">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="lab_l_35">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="lab_l_25">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLabel" name="lab_l_6">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lab_l_15">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="QLabel" name="lab_l_65">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="16">
    <widget class="QLabel" name="lab_l_8">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="lab_r_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="lab_l_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lab_l_1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QLabel" name="lab_r_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lab_r_1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="15">
    <widget class="QLabel" name="lab_l_75">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <widget class="QLabel" name="lab_l_85">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="14">
    <widget class="QLabel" name="lab_l_7">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lab_l_05">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="19">
    <widget class="QLabel" name="lab_r_95">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="17">
    <widget class="QLabel" name="lab_r_85">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="15">
    <widget class="QLabel" name="lab_r_75">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QLabel" name="lab_r_65">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="11">
    <widget class="QLabel" name="lab_r_55">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
	background: rgb(50, 50, 0);
}

QLabel::enabled{
	background: rgb(216, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QLabel" name="lab_r_45">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="lab_r_35">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="lab_r_25">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lab_r_15">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lab_r_05">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border-radius: 3px;
}

QLabel::disabled{
	background: rgb(0, 50, 0);
}

QLabel::enabled{
	background: rgb(0, 216, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="19">
    <widget class="QLabel" name="lab_l_95">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border-radius: 3px;
}

QLabel::disabled{
background: rgb(50, 0, 0);
}

QLabel::enabled{
background: rgb(192, 0, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayerPlugin</class>
   <extends>QWidget</extends>
   <header>PlayerPlugin/PlayerPlugin.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
